lib/reporters/mini.js

```diff
@@ -11,9 +11,9 @@ var repeating = require('repeating');
 var colors = require('../colors');
 
 chalk.enabled = true;
-Object.keys(colors).forEach(function (key) {
+for (let key of Object.keys(colors)) {
 	colors[key].enabled = true;
-});
+}
```

Comment:
Why are we setting this both here and in the constructor? Wouldn't it make more sense to only set it in the constructor?

Something like this:

```js
chalk.enabled = this.options.color;
for (const key of Object.keys(colors)) {
	colors[key].enabled = this.options.color;
}
```